src/viam/sdk/robot/client.cpp

```diff
@@ -191,7 +191,7 @@ void RobotClient::refresh() {
     }
     bool is_equal = current_resources.size() == resource_names_.size();
     if (is_equal) {
-        for (int i = 0; i < current_resources.size(); ++i) {
+        for (long unsigned int i = 0; i < resource_names_.size(); ++i) {
```

Comment:
@acmorrow I got compiler errors here when I used both `int` and `auto`. having `long unsigned int` here feels super wordy and a bit smelly, but I wasn't sure how to fix it. Any thoughts?

Reply:
Did you try `size_t`?